extensions/import/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -134,18 +144,32 @@ export class DataTierApplicationWizard {
 						break;
 					}
 				}
-			} else if (idx === 2) {
+			//deploy path has 4 pages, not 3 pages like the other operations
+			} else if ((idx === 2 && this.wizard.pages.length !== 4) || idx === 3) {
```

Comment:
This might get confusing since index 2 could be the summary page or it could be select deploy action page if the operation is deploy or generate script. 

I could check the operation instead of checking the page length, but that's longer since both deploy and generate script have page length 4.

Reply:
I was thinking each wizard page would have it's own enum: DacFxDeployWizardPages, DacFxExtractWizardPages, etc. So, depending on the wizard type, you can use the enums specific to the page.

The goal is to make this code more readable and maintainable as we are going to be adding pages, so think about other approaches if my suggestion doesn't make sense, or moves us away from our goal.

Reply:
ok I added more enums to make the checks more clear